dev/make-distribution.sh

```diff
@@ -123,26 +123,26 @@ fi
 if [ ! "$(command -v "$MVN")" ] ; then
     echo -e "Could not locate Maven command: '$MVN'."
     echo -e "Specify the Maven command with the --mvn flag"
-    exit -1;
+    exit 255;
```

Comment:
SC2242 (error): Can only exit with status 0-255. Other data should be written to stdout/stderr.